PyInstaller/loader/pyimod03_importers.py

```diff
@@ -642,9 +656,13 @@ class CExtensionImporter(object):
         full.module.name.cpython-33m.so
         full.module.name.abi3.so
     """
+
     def __init__(self):
         # Cache directory content for faster module lookup without
         # file system access.
+
+        # TODO: need recursive read capability
+
```

Comment:
We cannot use os.path.walk here. I am not sure how to build up recursive capability.